dt/uni05epsilon/edpm/nodeset/nova_custom.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nova-custom-config
+data:
+  25-nova-custom.conf: |
+    # NOTE(gibi): We need to disable multipath as IPv6 is not fully
+    # working in this job with NetApp.
+    # Enable multipath when OSPRH-7393 is resolved.
+    [libvirt]
+    volume_use_multipath = False
```

Comment:
form a nova prespective multipath is enabeld by default and tested in alpha so i think this is ok to disabel for now form a compute perspective.

we need to confirm if we actually have multiple paths in the alpha job to validate the cidner/storage backend part of that however in the context of this job i think this is fine to proceed with.